config/locales/en.yml

```
@@ -20,3 +21,7 @@ en:
           attributes:
             free_of_sanctions:
               blank: Tell us if your employment record is free of sanctions
+        teach_children_form:
+          attributes:
+            teach_children:
+              inclusion: Tell us if you are qualified to teach children
```

I think `qualified` is a bit of a loaded word:

```suggestion
              inclusion: Tell us if you have experience with teaching children
```